src/index.js

```diff
@@ -5,7 +5,6 @@ addNewList();
 addToArray();
 displayTasks();
 removeTask(); */
```

Comment:
- [ ] Is possible to see that you are importing methods from a file that does not exist yet and are commenting on the code. Please do not comment code out, it makes your code messy and does not add functionality. Kindly remove the code not used.